Simulation.py

```diff
@@ -1,38 +1,157 @@
-import pygame
+
 from CVRPTW_chromosome import Chromosome
 from CVRPTW_info import CVRPTWInfo
 
+import pygame, colorsys, pygame_widgets
+from pygame_widgets.slider import Slider
+from pygame_widgets.textbox import TextBox
+from pygame_widgets.button import Button
+
 
 class Interface:
 
     screen_dimensions = (1920,1080)
 
-    white = (200, 200, 200)
+    white = (255, 255, 255)
```

Comment:
c mieux de les nommer en majuscule car c'est des constates
```suggestion
    WHITE = (255, 255, 255)
```